lib/commands/build.dart

```diff
@@ -31,7 +31,7 @@ class BuildTpkCommand extends BuildSubCommand with TizenExtension {
       'target-arch',
       splitCommas: true,
       defaultsTo: <String>['arm'],
-      allowed: <String>['arm', 'aarch64', 'x86'],
+      allowed: <String>['arm', 'arm64', 'x86'],
```

Comment:
Just curious, why replace aarch64 with arm64?

Reply:
Because

- It's more consistent (arm-arm64-x86) and easier to remember for users
- The Flutter engine uses the term (`flutter/tools/gn --linux-cpu arm64 ...`) and so does our CI

Reply:
Now we use "arm64" instead of "aarch64" in most cases (unless we're interacting with the Tizen CLI or building a .NET tpk).